realm/realm-library/src/main/java/io/realm/internal/SharedRealm.java

```diff
@@ -175,6 +192,16 @@ public boolean isInTransaction() {
         return nativeIsInTransaction(nativePtr);
     }
 
+    public void setSchemaVersion(long schemaVersion) {
+        // FIXME migrate to ObjectStore
```

Comment:
Indeed. The thought behind my initial comment was  that we need to either duplicate the method (adding it to object store's `SharedRealm`) or make it public in `ObjectStore`. Sorry for not making it clear.